helm/hpcc/templates/_helpers.tpl

```diff
@@ -526,16 +526,20 @@ securityContext:
   capabilities:
     add:
     - SYS_PTRACE
+  runAsNonRoot: true
+  runAsUser: 10000
+  runAsGroup: 10001
```

Comment:
why have these 3 lines been duplicated? 
They were already common / included unconditionally outside of the if block before this change.

Reply:
Sorry, I thought that they should be in order. I will fix.